processing/data_collection/gazette/spiders/ce_fortaleza.py

```diff
@@ -0,0 +1,55 @@
+import re
+import dateparser
+import w3lib.url
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+
+
+class CeFortalezaSpider(Spider):
+    GAZETTE_ELEMENT_CSS = '.diarios-oficiais .table-responsive tbody tr'
+    DATE_CSS = 'td:nth-child(2)::text'
+    EXTRA_CSS = 'td:nth-child(1)::text'
+    NEXT_PAGE_CSS = 'ul.pagination .page-link::attr(href)'
+
+    MUNICIPALITY_ID = '2304400'
+
+    allowed_domains = ['apps.fortaleza.ce.gov.br']
+    name = 'ce_fortaleza'
+    start_urls = ['http://apps.fortaleza.ce.gov.br/diariooficial/?mes-diario=todos']
+
+    def start_requests(self):
+        requests = []
+
+        for year in range(2015, datetime.now().year + 1):
+            year_url = w3lib.url.add_or_replace_parameter( self.start_urls[0], 'ano-diario', year)
+            requests.append(Request(year_url))
```

Comment:
You don't need to create a list of requests. Just yield each request as soon you define it:
`yield Request(year_url)`